setup.py

```diff
@@ -1,15 +1,15 @@
-'''Setup module for Rubin Observatory Argo Workflow dispatcher (Nublado).
-'''
+"""Setup module for Rubin Observatory Argo Workflow dispatcher (Nublado).
+"""
```

Comment:
I'm assuming the single to double quote changes are all from `black`.  I'd prefer non-code changes be in a different commit.